configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerDB.java

```diff
@@ -28,7 +28,7 @@
     public ConfigServerDB(ConfigserverConfig configserverConfig) {
         this.configserverConfig = configserverConfig;
         this.serverDB = new File(Defaults.getDefaults().underVespaHome(configserverConfig.configServerDBDir()));
-        create();
+        createDirectory(serverdefs());
         try {
             initialize(configserverConfig.configModelPluginDir());
```

Comment:
@hmusum For the TenantTest.testJsonSerialization() test, this initialize() call ends up trying to copy classes() "/home/hakon/1/vespa-engine/vespa/configserver/target/1509484982440-0" to serverdefs() "/home/hakon/1/vespa-engine/vespa/configserver/target/1509484982440-0/serverdef". Which ends up recursing down and creating as many serverdef as it possibly can before the OS returns an error.

I wouldn't have noticed this if it were not for the fact than mvn clean is extremely slow on configserver.

Reply:
Nice catch @hakonhall, thanks! PR to fix this: https://github.com/vespa-engine/vespa/pull/3958